sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -120,6 +120,7 @@
 import org.apache.beam.sdk.util.Reshuffle;
 import org.apache.beam.sdk.util.SystemDoFnInternal;
 import org.apache.beam.sdk.util.Transport;
+import org.apache.beam.sdk.util.WindowedValue;
```

Comment:
We don't want to expose `WindowedValues` to users directly - it's the compressed representation that doesn't really exist for Pipeline authors.

We've added a `ValueInSingleWindow` in the testing package, which could be pulled up to the `value` package and used here instead.